apps/frontend/src/api/dashboard.ts

```diff
@@ -1,12 +1,5 @@
-import useFetch from 'use-http';
 import { DashboardResponse } from '@asap-hub/model';
 
-import { API_BASE_URL } from '../config';
-import { useFetchOptions } from './util';
+import { useGetOne } from './get-one';
 
-export const useDashboard = () =>
-  useFetch<DashboardResponse>(
-    `${API_BASE_URL}/dashboard`,
-    useFetchOptions(),
-    [],
-  );
+export const useDashboard = () => useGetOne<DashboardResponse>(`dashboard`);
```

Comment:
Thanks for changing it here as well! We should also adapt the FE `Dashboard` component to use the new structure of `GetOne`-type components. But doesn't need to be in this PR and doesn't need to be you who does it

Reply:
Nvm, you did that as well! Thanks :tada: